README_CN.md

```diff
@@ -0,0 +1,621 @@
+            <td>选择与使用的乐鑫设备目标相匹配的 OpenOCD 配置文件。例如，可以选择 DevKitC 或 ESP-Wrover-Kit。使用 JTAG 接口进行烧录或对设备进行调试时，此步骤必不可缺。</td>
+            <td></td>
+            <td></td>
+        </tr>
+        <tr>
+            <td>构建、烧录项目并监视设备</td>
+            <td>此命令可用于构建项目、将二进制程序写入设备，并启动监视终端，效果类似于 <strong>idf.py build flash monitor</strong>。</td>
+            <td><kbd>⌘</kbd> <kbd>I</kbd> <kbd>D</kbd></td>
+            <td><kbd>Ctrl</kbd> <kbd>E</kbd> <kbd>D</kbd></td>
+        </tr>
+        <tr>
+            <td rowspan=5 align="center">创建项目</td>
+            <td>展示示例项目</td>
+            <td>启动 UI 以显示所选框架的示例，可从中创建新项目。此命令将显示扩展中已配置的框架，如果想查看 ESP-Rainmaker 示例，需要先运行 <strong>安装 ESP-Rainmaker</strong> 命令（或设置相应的 idf.espRainmakerPath），然后执行此命令以查看示例。</td>
+            <td></td>
+            <td></td>
+        </tr>
+        <tr>
+            <td>基于模板创建新项目</td>
+            <td>使用扩展中的项目模板来创建一个新的 ESP-IDF 项目。</td>
+            <td><kbd>⌘</kbd> <kbd>I</kbd> <kbd>C</kbd></td>
+            <td><kbd>Ctrl</kbd> <kbd>E</kbd> <kbd>C</kbd></td>
+        </tr>
+        <tr>
+            <td>创建新 ESP-IDF 组件</td>
+            <td>在当前目录下，基于 ESP-IDF 组件模板创建新组件。</td>
+            <td></td>
+            <td></td>
+        </tr>
+        <tr>
+            <td>导入 ESP-IDF 项目</td>
+            <td>导入现有的 ESP-IDF 项目，在新位置添加 .vscode 和 .devcontainer 文件，同时可以重命名项目。</td>
+            <td></td>
+            <td></td>
+        </tr>
+        <tr>
+            <td>新项目</td>
+            <td>启动 UI，通过 ESP-IDF 项目创建向导，使用 ESP-IDF 中的示例模板和扩展中配置的其他框架。</td>
+            <td><kbd>⌘</kbd> <kbd>I</kbd> <kbd>N</kbd></td>
+            <td><kbd>Ctrl</kbd> <kbd>E</kbd> <kbd>N</kbd></td>
+        </tr>
+        <tr>
+            <td rowspan=7 align="center">烧录</td>
+            <td>选择烧录方式</td>
+            <td>选择用于 <strong>烧录项目</strong> 命令的烧录方法，可选择 DFU、JTAG 或 UART 接口。</td>
+            <td></td>
+            <td></td>
+        </tr>
+        <tr>
+            <td>烧录项目</td>
+            <td>将当前项目生成的二进制文件烧录至目标设备。此命令将根据 <strong>idf.flashType</strong> 使用 UART、DFU 或 JTAG 接口。</td>
+            <td><kbd>⌘</kbd> <kbd>I</kbd> <kbd>F</kbd></td>
+            <td><kbd>Ctrl</kbd> <kbd>E</kbd> <kbd>F</kbd></td>
+        </tr>
+        <tr>
+            <td>通过 DFU 接口烧录项目</td>
+            <td>通过 DFU，将当前 ESP-IDF 项目的二进制文件写入 flash 芯片，此方案仅适用于 ESP32-S2 和 ESP32-S3。</td>
+            <td></td>
+            <td></td>
+        </tr>
+        <tr>
+            <td>通过 UART 接口烧录项目</td>
+            <td>通过 esptool.py，将当前 ESP-IDF 项目的二进制文件写入 flash 芯片。</td>
+            <td></td>
+            <td></td>
+        </tr>
+        <tr>
+            <td>通过 JTAG 接口烧录项目</td>
+            <td>使通过 OpenOCD JTAG，将当前 ESP-IDF 项目的二进制文件写入 flash 芯片。</td>
+            <td></td>
+            <td></td>
+        </tr>
+        <tr>
+            <td>加密并烧录项目</td>
+            <td>执行项目烧录，同时为需要加密的分区添加 <strong>--encrypt</strong>。</td>
+            <td></td>
+            <td></td>
+        </tr>
+        <tr>
+            <td>擦除设备 flash 数据</td>
```

Comment:
This command appears twice (also in line 485), pls confirm and update for en and cn if necessary.

Reply:
Hi @April-Yjj. Yes, I also noticed this during translation. Actually, if we go through the whole table carefully, we can see that besides this command, several other IDF extension commands appear twice or even three times. I believe the author just intended to ensure that each category includes all related commands and their descriptions. Considering this doesn't hinder readability for users, I suggest keeping them as they are. Thx!